contracts/Vault.sol

```diff
@@ -35,12 +45,32 @@ contract Vault is AccessControl {
         grantRole(PORTFOLIO_MANAGER, _portfolioManager);
     }
 
+    function setRewardManager(address _rewardManager) public onlyAdmin {
+        require(_rewardManager != address(0), "Zero address not allowed");
+        grantRole(REWARD_MANAGER, _rewardManager);
+    }
+
+
+    function setAaveReward(address _aaveReward) public onlyAdmin {
+        require(_aaveReward != address(0), "Zero address not allowed");
+        aaveReward = IAaveIncentivesController(_aaveReward);
+    }
+
+
+
     //TODO: do we really need this feature?
     function removePortfolioManager(address _portfolioManager) public onlyAdmin {
         require(_portfolioManager != address(0), "Zero address not allowed");
         revokeRole(PORTFOLIO_MANAGER, _portfolioManager);
     }
 
+
+    function claimRewardAave(address[] calldata assets) public onlyRewardManager {
+        uint256 balance = aaveReward.getRewardsBalance(assets, address(this));
+        aaveReward.claimRewards(assets, balance, address(this));
```

Comment:
No need call getRewardsBalance because in current usage it same to
work same as aaveReward.claimRewards(assets, uint256.max, address(this)); because of https://github.com/aave/aave-stake-v2/blob/master/contracts/stake/AaveIncentivesController.sol#L140
but execute code for getRewardsBalance twice